pkg/universal-cnf/vppagent/backend_test.go

```diff
@@ -18,6 +18,7 @@ const (
 	dstIpAddrClient = "192.168.22.2"
 	dstIpRouteClient =  "192.168.0.0/16"
 	ifName = "endpoint0"
+	mechanismType = "MEMIF"
```

Comment:
seems like the fixes in this file are unrelated to the WCM connector NSE work, right?  Are they more appropriate in another PR ?

Reply:
Sure, I'll move these changes into a separate PR. Thanks for the review Tim!!! :)